src/pk_core.c

```diff
@@ -1122,6 +1122,20 @@ Var varBitAnd(PKVM* vm, Var v1, Var v2) {
   return VAR_NULL;
 }
 
+Var varBitOr(PKVM* vm, Var v1, Var v2) {
+
+  int64_t i1, i2;
+  if (isInteger(v1, &i1)) {
+    if (validateInteger(vm, v2, &i2, "Right operand")) {
+      return VAR_NUM(i1 | i2);
```

Comment:
This cause a warning in msvc [C4244](https://docs.microsoft.com/en-us/cpp/error-messages/compiler-warnings/compiler-warning-levels-3-and-4-c4244?view=msvc-160). could you cast it to `double` before calling `VAR_NUM` also for `VAR_NUM(i1 & i2)`

```diff
-      return VAR_NUM(i1 | i2);
+      return VAR_NUM((double)(i1 | i2));
```